instructors/instructor-notes.md

```diff
@@ -69,6 +69,21 @@ To prevent the learners from having to re-enter their password each time they pu
 $ git config --global credential.helper 'cache --timeout=10000000'
 ```
 
+## RStudio Color Preview
+
+RStudio has a feature to preview the color for certain named colors and hexadecimal colors. This may confuse or distract learners (and instructors) who are not expecting it.
+
+Mainly, this is likely to come up during the episode on "Data Structures" with the following code block:
+
+```r
+cats <- data.frame(coat = c("calico", "black", "tabby"),
+                    weight = c(2.1, 5.0, 3.2),
+                    likes_string = c(1, 0, 1))
+```
+
+This option can be turned off an on in the following menu setting:
```

Comment:
minor typo here "turned off and **on**"

Reply:
updated in [`6dab42c` (#871)](https://github.com/swcarpentry/r-novice-gapminder/pull/871/commits/6dab42cb895d37c4a3663d84c8de20752af690ee)